src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -30,6 +30,29 @@ using StringInputs = std::variant<std::string, std::vector<std::string>>;
 */
 static constexpr ov::Property<SchedulerConfig> scheduler_config{"scheduler_config"};
 
+/**
+* @brief ModelDesc serves to activate speculative decoding model in continuous batching pipeline.
+* Create LLMPipeline and fill it with sutable values.
+* In case of empty device speculative decoding will be run with the same configuration as main_pipeline.
+*/
+struct ModelDesc {
+    std::string model_path;
+    std::string device;
+    ov::genai::SchedulerConfig scheduler_config;
+    ov::AnyMap plugin_config;
+
+    ModelDesc(const std::string& model_path,
+              const std::string& device = "",
+              const ov::AnyMap& plugin_config = {},
+              const ov::genai::SchedulerConfig& scheduler_config = {}) :
```

Comment:
Similarly to `LLMPipeline` ctors, `scheduler_config` doesn't need to be passed as an explicit parameter, it could be a property if we add "property friendly" ctor for ModelDesc.

Reply:
Implemented

Reply:
@slyalin According to some technical issues will be implemented in next PRs